scripts/generate_easystacks_for_existing_stack/stack_to_json_parallel.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Usage: ./script.sh [MAX_JOBS]
+MAX_JOBS=${1:-4}  # Default to 4 concurrent jobs if not specified
+DEBUG=0
+BASE_STACK=/cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software
+EB_BOOTSTRAP=5.0.0
+TMPDIR=$(mktemp -d)
+
+declare -A gcc_to_foss=( ["12.2.0"]="2022b" ["12.3.0"]="2023a" ["13.2.0"]="2023b" )
+
+if [[ ! -d ${BASE_STACK} ]]; then
+  echo "The given base stack (${BASE_STACK}) is not a directory."
+  exit 1
+fi
+
+apps=$(find ${BASE_STACK} -mindepth 2 -maxdepth 2 -type d)
+
+# Job limiter
+job_count=0
+run_limited() {
+  ((job_count++))
+  if (( job_count >= MAX_JOBS )); then
+    wait -n  # wait for one job to finish
+    ((job_count--))
+  fi
+}
+
+for app_dir in $apps; do
+run_limited
+(
+  app_version=$(basename "${app_dir}")
+  app_name=$(basename "$(dirname "${app_dir}")")
+
+  if [[ ${app_name} == "EESSI-extend" ]]; then
+    exit 0
+  fi
+
+  easyblocks=${app_dir}/easybuild/reprod/easyblocks/*.py
+  easyconfig=${app_dir}/easybuild/reprod/${app_name}-${app_version}.eb
```

Comment:
We shouldn't use the easyconfig from the `reprod` dir, that's causing too many issues with missing parameters/patches.
```suggestion
  easyconfig=${app_dir}/easybuild/${app_name}-${app_version}.eb
```